src/main/java/Storage/Storage.java

```diff
@@ -16,15 +16,15 @@
 public class Storage {
 
     private String fileName;
-    private boolean appendToFile = false;
+    private boolean canAppendToFile = false;
 
     public Storage(String fileName) {
         this.fileName = fileName;
     }
 
     public Storage(String fileName, boolean appendToFile) {
         this.fileName = fileName;
-        this.appendToFile = appendToFile;
+        this.canAppendToFile = appendToFile;
     }
```

Comment:
Maybe certain parts of this code can also be extracted out to improve readability?